Running into panic when trying to install bin to be self managed

Hi there,

I downloaded bin from the releases page on my System (Arm based) - and tried the install command to let bin manage itself:

```bash
   • Getting latest release for marcosnils/bin
   • Starting download of https://api.github.com/repos/marcosnils/bin/releases/assets/164207200
15.89 MiB / 15.89 MiB [---------------------------------------------------------------------------------------------] 100.00% 14.18 MiB p/s 1s
   • Copying for bin_0.17.5_darwin_arm64@v0.17.5 into /Users/ista-alexanderpadberg/.local/bin
panic: assignment to entry in nil map

goroutine 1 [running]:
github.com/marcosnils/bin/pkg/config.UpsertBinary(0x14000440150)
        /home/marcos/Projects/bin/pkg/config/config.go:109 +0x3c
github.com/marcosnils/bin/cmd.newInstallCmd.func1(0x14000147900?, {0x140003cd260?, 0x4?, 0x1048f530f?})
        /home/marcos/Projects/bin/cmd/install.go:75 +0x318
github.com/spf13/cobra.(*Command).execute(0x14000152900, {0x140003cd230, 0x1, 0x1})
        /home/marcos/Projects/gopath/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:916 +0x66c
github.com/spf13/cobra.(*Command).ExecuteC(0x14000152600)
        /home/marcos/Projects/gopath/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x320
github.com/spf13/cobra.(*Command).Execute(...)
        /home/marcos/Projects/gopath/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/marcosnils/bin/cmd.(*rootCmd).Execute(0x1400000d2f0, {0x1400001e1c0, 0x2, 0x140003cd030?})
        /home/marcos/Projects/bin/cmd/root.go:36 +0x144
github.com/marcosnils/bin/cmd.Execute({0x1400006a1c0, 0x6b}, 0x104a24ac0?, {0x1400001e1c0, 0x2, 0x2})
        /home/marcos/Projects/bin/cmd/root.go:26 +0xc0
main.main()
        /home/marcos/Projects/bin/main.go:21 +0xa0
```

As I am guessing, this has something to do with my config (I have a second system (Linux)) that I share my dot files with:

File: ~/.config/bin/config.json
```json 
   {
        "default_path": "$HOME/.local/bin"
   }
```

Any help is much appreciated, if you need more information, please let me know!

--- a/pkg/config/config.go
+++ b/pkg/config/config.go
@@ -94,6 +94,11 @@ func CheckAndLoad() error {
 		}
 
 	}
+
+	if cfg.Bins == nil {
+		cfg.Bins = map[string]*Binary{}
+	}
+
 	log.Debugf("Download path set to %s", cfg.DefaultPath)
 	return nil
 }

